teps/0084-endtoend-provenance-collection.md

```diff
@@ -237,6 +545,31 @@ via CLI, dashboard or a monitoring system.
 Consider including folks that also work on CLI and dashboard.
 -->
 
+<!-- TODO: Revisit this once event-payload is taken into consideration -->
+
+<!-- TODO: Is there a CLI plugin for chains? -->
```

Comment:
Not as far as I know - though I can see this being useful for getting / verifying attestations!

Reply:
There's a [tkn chain](https://github.com/tektoncd/cli/blob/a71c2191a42b0c50faffb1b94406e12f188476e1/docs/cmd/tkn_chain.md) command which you can use for getting payload/signatures. You could add in something for attestations as well!